jenkins/job-configs/kubernetes-jenkins/credentials.yaml

```diff
@@ -0,0 +1,19 @@
+- wrapper:
```

Comment:
I thought about that, but I wasn't sure that a comment would be any more descriptive than the env var name itself.